forward_engineering/utils/getClusteringKey.js

```diff
@@ -0,0 +1,39 @@
+const mapName = ({ name }) => `"${name}"`;
```

Comment:
since it is only adding a quotation marks, can't you reuse the already existing method `addQuotes` in snowflakeHelper?